test/e2e_node/jenkins/gci-init.yaml

```diff
@@ -1,10 +1,9 @@
 #cloud-config
 
 runcmd:
-  - mount /tmp /tmp -o remount,exec,suid
-  - ETCD_VERSION=v2.2.5
-  - curl -L https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -o /tmp/etcd.tar.gz
+  - mount tmpfs /tmp -o remount,exec,suid
```

Comment:
Just for my education. What is the difference between `mount /tmp /tmp` and `mount tmpfs /tmp`? GCI is not using tmpfs by default?